src/makeCalculator.js

```diff
@@ -36,8 +36,43 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    operate: function(operation, number) {
+      operation(number);
+
+      return this;
+    },
+    add: function(number) {
+      calc.result += number;
+
+      return this;
+    },
+    subtract: function(number) {
+      calc.result -= number;
```

Comment:
Same here as above:

Instead of using `calc.result` use `this.result`. `this` keyword refers to the current object, providing flexibility if the object name is changed.

Please do that in all functions :)